pinc/RoundDescriptor.inc

```diff
@@ -199,6 +199,25 @@ class Round extends Stage
         // Give them the lowest honorific.
         return $honorific;
     }
+
+    // -----------
+
+    public function validate_user_can_access()
```

Comment:
We aren't very consistent in the codebase with this, but I would prefer that this either take the user we are validating, or we use `validate_current_user_can_access()` to be clear that it's `$pguser` being used. Ideally the former as that's more extensible.